### ISSUE TITLE
Value as concept for Measurement should not be a concept picker, but conceptset

### ISSUE THREAD

**Issue by cgreich** [OHDSI/Atlas#2161]

**Comment by Alexdavv** [comment]
The same for Observation, please.


BTW, @cgreich, @pbr6cornell, I've looked through all the attributes. Don't we need concept-set functionality for the following attributes (when working inside the initial event, restriction creteria or inclusion criteria):
- Visit / Visit Occurrence what must be the same thing (is it even used in such a way? If yes, hierarchy will not gonna work here now)
![image](https://user-images.githubusercontent.com/32357692/77844023-53f2f880-71ab-11ea-917e-9d07e97c1834.png)
![image](https://user-images.githubusercontent.com/32357692/77843623-99152b80-71a7-11ea-909b-25c10621fda1.png)
- Provider Specialty (hierarchy will not gonna work here now)
![image](https://user-images.githubusercontent.com/32357692/77843935-88b28000-71aa-11ea-89e2-4c3e08c5d9a5.png)
- Race
- Event Type (even after type becomes Domain-unspecific)
- Place of Service (is it still supported?)
- Anatomic site in Specimen

**And we should be ok with [import button working](https://github.com/OHDSI/Atlas/issues/2160)** for the rest:
- Gender
- Ethnicity
- Unit
- Route
- Value Operator
- Observation Qualifier
- Procedure Modifier
- Disease status in Specimen


**And finally, additional filters (by vocabulary_id and standard_concept) might be really helpful in picker window:**
![image](https://user-images.githubusercontent.com/32357692/77844425-57887e80-71af-11ea-879c-3f6abc163293.png)

**Comment by anthonysena** [comment]
Tagging @chrisknoll here for input but at first glance the challenge I see is the JSON cohort expression is expecting a list of concept IDs vs a concept set reference. Changing this paradigm would be a breaking change and would better be reserved for discussion with the next major release. 

As for the concept picker window this again may be better for the 3.0 version.

**Comment by chrisknoll** [comment]
Agreed, this is a breaking change if we re-define what those fields are, but we can introduce new criteria params that are stored as a different field and are understood to be concept set instead of list of concepts.

**Comment by anthonysena** [comment]
Also noting that @clairblacketer raised a similar point in #2140 as it relates to using the visit hierarchy as part of defining visit events.

**Comment by Alexdavv** [comment]
> related to other issues. if we replace all 'concept pickers' with conceptsets we will close mutiple issues.

Is it a good solution for the things where the concept-set would always include 1-2 concepts?



> **And we should be ok with [import button working](https://github.com/OHDSI/Atlas/issues/2160)** for the rest:
> 
> * Gender
> * Ethnicity
> * Unit
> * Route
> * Value Operator
> * Observation Qualifier
> * Procedure Modifier
> * Disease status in Specimen

**Comment by MaximMoinat** [comment]
@chrisknoll Is this a feature that is still being considered for future version of Atlas?

It seems that this is already implemented for place of service, so might be copy-able to the other concept pickers?

<img width="501" height="92" alt="Image" src="https://github.com/user-attachments/assets/537d2296-65e9-4b2d-82f6-a44dca5b8477" />

**Comment by chrisknoll** [comment]
Yep, we can plan on applying it to other places.  We'll leave the existing 'list of concept IDs' in place for certain things for backwards compatabiltiy, but we can push on covering remaining areas that only have the concept-list way of specifying a concept list.
